src/VisualStudio/Core/Def/Implementation/CodeCleanup/CommonCodeCleanUpFixerDiagnosticIds.cs

```diff
@@ -12,115 +12,322 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
 {
     internal static class CommonCodeCleanUpFixerDiagnosticIds
     {
+        [Export]
+        [FixId(AbstractCodeCleanUpFixer.RemoveUnusedImportsFixId)]
+        [Name(AbstractCodeCleanUpFixer.RemoveUnusedImportsFixId)]
+        [Order(After = AbstractCodeCleanUpFixer.FormatDocumentFixId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://docs.microsoft.com/visualstudio/ide/reference/options-text-editor-csharp-advanced#using-directives")]
+        [ExportMetadata("EnableByDefault", true)]
+        [LocalizedName(typeof(ServicesVSResources), nameof(FeaturesResources.Remove_unnecessary_Imports_or_usings))]
+        public static readonly FixIdDefinition? RemoveUnusedImports;
```

Comment:
📝 The reordering here made the review very difficult

Reply:
let me fix up the diff so they are just additions at the end instead of sorted alphabetically.